### DISCUSSION TITLE
Coding apa yang digunakan seperti Unfilter 'blanks' pada Excel ?

### DISCUSSION THREAD

**Discussion by vlvivian** [teamalgoritma/community#70] [Q&A] [answered]
Pak/Bu mau bertanya pakai Coding apa jika seperti di Excel kita mau Unfilter 1 kolom yang 'blanks' ?

**Comment by onlyphantom** [comment, marked as answer]
If I'd like to get rows of data where `temperature` is blank (`NA`), I'd just combine the `is.na()` method with a `subset`. 

```
subset(df, is.na(df$temperature))
```
This returns all rows of `df` where `temperature` is blank / empty.

**Reply by vlvivian** [reply]
Terima kasih pak.
Mau bertanya lagi Pak bagaimana jika ingin menghapus baris yang NA ?

**Reply by onlyphantom** [reply]
df <- df[complete.cases(df), ]

or:

df <- subset(df, complete.cases(df))

😀
